### DISCUSSION TITLE
Welcome to Olshot Discussions!

### DISCUSSION THREAD

**Discussion by olsson82** [olsson82/olshot#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by helidave** [comment]
Excellent news!  I built the last 2 versions of this game and it was really great.  I had a few issues and I would change a couple of things if I could but I'm excited to see what you come up with this time.  I added a simple turning target system using an Arduino, a bluetooth module and a car wiper motor as this gave me a system similar to the professional one I use at my shooting club.  It was very amateurish but it worked.  I controlled it with an App I wrote (very badly) for my phone which again, simulated the times the club system used.  My only problem was making the targets in such a way that I didn't get lead everywhere.  I tried a BB type gun because these have a similar action to the gun I use at the club but the BB's go everywhere so I gave up on that.

**Reply by olsson82** [reply]
Thanks, im glad to do this again, and i think this will be mutch better this time with prebuilt pcb.

I was thinking on making an app and use bluetooth, but never done an app and then i need to learn for both ios and android, so im thinking on use nextion touch display instead.

Don't know the weights of the targets, but you could use server motors, they are fast, and there are strong ones.

If you have something you think i have missed, let me know.

**Reply by helidave** [reply]
Pre-built pub is certainly better both for reliability and installation for sure.  As soon as you confirm the pub design files I will order some from my usual supplier.  For App development I used MIT App Inventor but I am not a software engineer so I needed something suitable for my skill set and it worked well, especially the bluetooth connection code.  It certainly made it easy to experiment with different ideas.  I have quite a few spare servo motors as I used to fly quite large RC Helicopters when I lived in the UK.  My problem was more choosing the right gun with the right ammunition (BB or Pellet) that replicated what I was doing at the club and mounting the targets in such a way that the sensor didn't get hit by the projectile but still sensed the hit.  I prefer the pellet for accuracy but it hits quite hard and the BB's are difficult to stop from ricocheting all over the place.

**Reply by helidave** [reply]
If possible, I would also like the code to remember previous players so I don't have to enter the names after every game.  Certainly, as the main user it would be nice if the system remembered me and assumed I would always be a player unless otherwise instructed.
If the system could have an o/p to control a motor to allow turning targets that would be fantastic.  Also, if the system could have means whereby the user tells it the number of targets for a given game rather than look for all the targets that would be useful as for example, if I was playing on my own I might just want a single turning target or 4 fixed targets at random.  Not sure if I have explained this very well so apologies.  Also, I am not expecting you to do any of this I realise you have your own requirements etc. but, as you asked for ideas!  Anyway, thanks again - fun project.

**Reply by olsson82** [reply]
Today im going to work on the main unit design :) But im going to do some changes on the target also. Going to change so i don't need to solder the lipo battery. I forgot that they have a contact on them. Also i can make some free space to digital and analoge for some custom add on the targets. My goal is also design some case for the target sensor, for 3d printing.

My plan is to have the system remember stuff. On the main unit im going to built in a sd card reader (also a real time clock), the other thing was not so bad idea. The nextion display is good that you can design them in a software so you can send data to the arduino from the touch screens.

When i have confirm that the pcb work i will release a bom list with every component. Also the files if someone want to modify them.
